src/common/common.cpp

```diff
@@ -290,3 +291,10 @@ std::wstring get_process_path(HWND window) noexcept {
   }
   return name;
 }
+
+std::wstring get_product_version() {
+  std::wstring version = L"" + std::to_wstring(VERSION_MAJOR) +

```

Comment:
`static` here would be great. What is the first `L""` for?

Reply:
L"" a leftover, will remove it, thanks.